app/models/observation_field_value.rb

```diff
@@ -96,12 +94,11 @@ def strip_value
   end
 
   def set_user
-    if updater_user_id
-      self.user_id ||= updater_user_id
-      self.updater_id = updater_user_id
+    if updater
+      self.user ||= updater
     elsif observation
-      self.user_id ||= observation.user_id
-      self.updater_id ||= user_id
+      self.user ||= observation.user
+      self.updater ||= user
```

Comment:
Noticed this when working on OFV, probably pre-dating `has_updater` inclusion. Seems like a fairly easy clean up. This and the associated controller params are the only non-spec changes here.

Reply:
Good catch.